libcloudforensics/providers/gcp/internal/compute_base_resource.py

```diff
@@ -76,7 +84,7 @@ def GetValue(self, key: str) -> str:
     """
 
     # pylint: disable=no-member
-    self._data = self.GetOperation()  # type: ignore
+    self._data = self.GetOperation()   # type: ignore
```

Comment:
Do we still need the pylint directive `no-member` and mypy `ignore` with the newly defined abstract method line 68?